src/test/kotlin/com/expedia/graphql/schema/extensions/AnnotationExtensionsTest.kt

```diff
@@ -0,0 +1,69 @@
+package com.expedia.graphql.schema.extensions
+
+import com.expedia.graphql.annotations.GraphQLDescription
+import com.expedia.graphql.annotations.GraphQLID
+import com.expedia.graphql.annotations.GraphQLIgnore
+import kotlin.reflect.full.declaredMemberProperties
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertNull
+import kotlin.test.assertTrue
+
+internal class AnnotationExtensionsTest {
+
+    @GraphQLDescription("class description")
+    @Deprecated("class deprecated")
+    @GraphQLIgnore
+    private data class WithAnnotations(
+        @Deprecated("property deprecated")
+        @property:GraphQLDescription("property description")
```

Comment:
Why is the `property` prefix used here? I think it's works without it?

Reply:
It does not work on data class properties. See the notes here: https://github.com/ExpediaDotCom/graphql-kotlin/wiki/@GraphQLDescription